MeshLib/IO/MPI_IO/NodePartitionedMeshReader.cpp

```diff
@@ -198,14 +198,26 @@ MeshLib::NodePartitionedMesh* NodePartitionedMeshReader::readBinary(
 MeshLib::Properties NodePartitionedMeshReader::readPropertiesBinary(
     const std::string& file_name_base) const
 {
-    const std::string fname_cfg = file_name_base +
-                                  "_partitioned_properties_cfg" +
+    MeshLib::Properties p;
+    readPropertiesBinary(file_name_base, MeshLib::MeshItemType::Node, p);
+    readPropertiesBinary(file_name_base, MeshLib::MeshItemType::Cell, p);
+    return p;
+}
+
+void NodePartitionedMeshReader::readPropertiesBinary(
+    const std::string& file_name_base, MeshLib::MeshItemType t,
+    MeshLib::Properties& p) const
+{
+    std::string const item_type =
+        t == MeshLib::MeshItemType::Node ? "node" : "cell";
+    const std::string fname_cfg = file_name_base + "_partitioned_" + item_type +
+                                  "_properties_cfg" +
                                   std::to_string(_mpi_comm_size) + ".bin";
     std::ifstream is(fname_cfg.c_str(), std::ios::binary | std::ios::in);
     if (!is)
     {
         WARN("Could not open file '%s' in binary mode.", fname_cfg.c_str());
-        return MeshLib::Properties();
+        return;
```

Comment:
ogs should not crash when the simulation does not need (cell or node) properties.